src/Symfony/Component/Validator/Constraints/Date.php

```diff
@@ -41,11 +41,7 @@ class Date extends Constraint
     #[HasNamedArguments]
     public function __construct(?array $options = null, ?string $message = null, ?array $groups = null, mixed $payload = null)
     {
-        if (\is_array($options)) {
-            trigger_deprecation('symfony/validator', '7.3', 'Passing an array of options to configure the "%s" constraint is deprecated, use named arguments instead.', static::class);
```

Comment:
in cases like this, we still have a legacy $options argument
would it make sense to try to remove it, using some BC layer?
or should we throw to be sure passing anything else than null forbidden?